components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -319,21 +351,18 @@ export default class AtMentionProvider extends Provider {
         this.updateMatches(resultCallback, this.items());
 
         // If we haven't gotten server-side results in 500 ms, add the loading indicator.
-        let showLoadingIndicator = setTimeout(() => {
+        let showLoadingIndicator: NodeJS.Timeout | null = setTimeout(() => {
             if (this.shouldCancelDispatch(prefix)) {
                 return;
             }
 
-            this.updateMatches(resultCallback, this.items().concat([{
-                type: Constants.MENTION_MORE_MEMBERS,
-                loading: true,
-            }]));
+            this.updateMatches(resultCallback, [...this.items(), {type: Constants.MENTION_MORE_MEMBERS, loading: true}]);
 
             showLoadingIndicator = null;
         }, 500);
 
         // Query the server for remote results to add to the local results.
-        this.autocompleteUsersInChannel(prefix).then(({data}) => {
+        this.autocompleteUsersInChannel(prefix).then(({data}: any) => {
```

Comment:
Is there another type we can use here instead of `any`?

Reply:
Fixed